config/config.ini.sample

```diff
@@ -0,0 +1,31 @@
+[library]
```

Comment:
Related to the comment on a README, put more comments, and possible even examples of values to be configured here.